bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/localConnectInverter.xml

```diff
@@ -0,0 +1,115 @@
+				<label>Inverter Input/Output Power</label>
+				<description>Power to/from the inverter</description>
+			</channel>
+			<channel id="inverter-current" typeId="system.electric-current">
+				<label>Inverter Input/Output Current</label>
+				<description>Current to/from the inverter</description>
+			</channel>
+			<channel id="inverter-voltage" typeId="system.electric-voltage">
+				<label>Inverter Voltage</label>
+				<description>Voltage of the inverter</description>
+			</channel>
+			<channel id="inverter-frequency" typeId="frequency">
+				<label>Inverter Voltage Frequency</label>
+				<description>Voltage frequency of the inverter.(should be the same as the feed in voltage frequency)</description>
+			</channel>
+
+			<channel id="pv1-voltage" typeId="system.electric-voltage">
+				<label>PV 1 Voltage</label>
+				<description>Electric voltage of PV String 1</description>
+			</channel>
+			<channel id="pv2-voltage" typeId="system.electric-voltage">
+				<label>PV 2 Voltage</label>
+				<description>Electric voltage of PV String 2</description>
+			</channel>
+			<channel id="pv1-current" typeId="system.electric-current">
+				<label>PV 1 Current</label>
+				<description>Electric current of PV String 1</description>
+			</channel>
+			<channel id="pv2-current" typeId="system.electric-current">
+				<label>PV 2 Current</label>
+				<description>Electric current of PV String 2</description>
+			</channel>
+			<channel id="pv1-power" typeId="system.electric-power">
+				<label>PV 1 Power</label>
+				<description>Electric power of PV String 1</description>
+			</channel>
+			<channel id="pv2-power" typeId="system.electric-power">
+				<label>PV 2 Power</label>
+				<description>Electric power of PV String 2</description>
+			</channel>
+			<channel id="pv-total-power" typeId="system.electric-power">
+				<label>PV Total Power</label>
+				<description>The sum of PV powers from all strings</description>
+			</channel>
+			<channel id="pv-total-current" typeId="system.electric-current">
+				<label>PV Total Current</label>
+				<description>The sum of PV currents from all strings</description>
+			</channel>
+
+			<channel id="battery-power" typeId="system.electric-power">
+				<label>Battery Power</label>
+				<description>Power to/from the battery</description>
+			</channel>
+			<channel id="battery-current" typeId="system.electric-current">
+				<label>Battery Current</label>
+				<description>Electric current to/from the battery</description>
+			</channel>
+			<channel id="battery-voltage" typeId="system.electric-voltage">
+				<label>Battery Voltage</label>
+				<description>Electric voltage of the battery</description>
+			</channel>
+			<channel id="battery-temperature" typeId="battery-temperature">
+				<label>Battery Temperature</label>
+				<description>Temperature of the battery</description>
+			</channel>
+			<channel id="battery-state-of-charge" typeId="system.battery-level">
```

Comment:
Perhaps this would be simpler (term mostly used in other bindings):
```suggestion
			<channel id="battery-level" typeId="system.battery-level">
```